.gitignore

```diff
@@ -9,3 +9,4 @@ tldextract.egg-info
 tldextract/.suffix_cache/*
 .tox
 .pytest_cache
+.coverage
```

Comment:
This project doesn't use any coverage tools, so this line is a little confusing. The line might belong in your personal .gitignore? I'm going to omit it from the PR. Although I would love coverage +/- tracking on PRs someday… 😄

Reply:
Agreed, it would be better in another future PR.